notice_and_comment/settings/base.py

```diff
@@ -18,3 +18,7 @@
     os.environ.get('VCAP_APP_PORT', '8000'))
 
 STATICFILES_DIRS = ['compiled']
+
+# Commenting
+BROKER_URL = 'redis://localhost:6379/0'
```

Comment:
This is the default local redis URL--what you get if you run `redis-server` with no options. I was thinking this would be a reasonable default for local development, and it's overridden in the production settings, where we expect to find a redis service provided by cloud.gov. Does that seem reasonable?